### PR TITLE
Mynewbranch

### PR DESCRIPTION
**Pull request by aashna15sethi**
added functionality to fizzbuzz

### CODE REVIEW

File: hw1/01-fizzbuzz.js [hunk 1]
```diff
@@ -11,7 +11,24 @@ Use console.log() to write the proper output to the command line.
 
 const fizzbuzz = () => {
   // Add your code here
-};
+  for (i = 1; i <= 100; i++)
+  {
+    if(i % 5 == 0 && i % 3 ==0){
+      console.log("fizzbuzz")
+    }
+    else if(i % 3 != 0 && i % 5 ==0){
```
Comment (Valbjornpdx) [inline hw1/01-fizzbuzz.js]: Make sure whitespace is consistent 
`    else if(i % 3 != 0 && i % 5 == 0){
`

**Comment by Valbjornpdx** [comment]
Functionality seems good to me! Just check the airbnb styling guide and match and go from there. Good job.

**Review by gaurikasar — APPROVED** [review]
Looks good to me

**Comment by JeremyIV** [comment]
A couple of comments not mentioned by other reviewers: your formatting is inconsistent, be sure to use an autoformatter like Prettier. Also, the `i % 3 != 0` is not required in the second condition: if execution makes it to this conditional, we already know the number isn't divisible by both 3 and 5.

**Pull request merged**